pkg/utils/webhook/add.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package webhook
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+// ServerConfig is a completed webhook server configuration.
+type ServerConfig struct {
+	// Name is the webhook server name
+	Name string
+	// URL is the URL that is used to register the webhooks in Kubernetes.
+	URL string
```

Comment:
I think the url is not needed as we generate the domain based on the service name and namespace